### DISCUSSION TITLE
can't download project

### DISCUSSION THREAD

**Discussion by Dron7347** [community/community#147687] [New to GitHub]
Hi, I can't download project, most of the files are missing in the .zip file, what should I do? This project https://github.com/MichaelMacha/SteamMultiplayerPeerExample/tree/demo

**Comment by Vinayak-YB** [comment]
Just clone it using `git clone https://github.com/MichaelMacha/SteamMultiplayerPeerExample.git` and build from source

**Comment by Rochdi7** [comment]
## Troubleshooting Missing Files in Downloaded .zip

If you are experiencing missing files when downloading a project from GitHub as a **.zip file**, try the following solutions:

### ✅ Clone the Repository Instead
Instead of downloading the ZIP, try cloning the repository directly using Git:

```bash
git clone --branch demo https://github.com/MichaelMacha/SteamMultiplayerPeerExample.git
```

This ensures you get the full project, including all branches.

### ✅ Check for Git LFS (Large File Storage)
If the project uses **Git LFS**, some files may not be included in the ZIP. To fetch these files, run:

```bash
git lfs install
git lfs pull
```

### ✅ Try a Different Browser
Sometimes, browser cache issues can affect downloads. Try using **Chrome, Firefox, or Edge** to see if that resolves the issue.

---
